content/blog/2018-11-11-colocr.md

```diff
@@ -0,0 +1,106 @@
+---
+slug: "colocr"
+title: Co-localization analysis of image microscopy
+package_version: 0.1.0
+authors:
+  - name: Mahmoud Ahmed
+    url: https://github.com/MahShaaban
+date: 2018-11-11
+categories: blog
+topicid:
+tags:
+- r
+- package
+- onboarding
+- imaging
+- reproducibility
+---
+
+A few months ago, I wasn't sure what to expect when reading microscopy images. I looked at the accompanying graph to understand the data or the point the authors is trying to make. Often, the graph represents one or more measures of the so-called co-localization, but I couldn’t figure out how to interpret them. It turned out; reading the images is simple. Cells are simultaneously stained by two dyes (say, red and green) for two different proteins. The color turns yellow in the merged image when the two proteins localize in proximity. 
```

Comment:
First two sentences might be clarified by saying you're looking at images in a published paper